server/controllers/stock/reports/stock/entry_purchase_receipt.js

```diff
@@ -42,22 +46,40 @@ async function stockEntryPurchaseReceipt(documentUuid, session, options) {
   `;
 
   const results = await Promise.all([
-    db.exec(sql, [db.bid(documentUuid)]),
+    db.exec(smSql, [db.bid(documentUuid)]),
     getVoucherReferenceForStockMovement(documentUuid),
   ]);
 
   const rows = results[0];
-  const voucherReference = results[1][0] ? results[1][0].voucher_reference : null;
-
   if (!rows.length) {
     throw new NotFound('document not found');
   }
 
+  const poUuid = rows[0].po_uuid;
+  const voucherReference = results[1][0] ? results[1][0].voucher_reference : null;
+
+  // Get the information about the original PO lots
+  const poSql = `
+     SELECT BUID(inventory_uuid) as inventory_uuid, quantity, unit_price
+     FROM purchase_item WHERE purchase_uuid = ?;
+     `;
+  const poLots = await db.exec(poSql, [db.bid(poUuid)]);
```

Comment:
Can we rename this variable?   Let's call it `poInventoryArticles`.  It took me a couple double takes thinking about lots rather than inventory articles, when these are actually inventory articles.

Reply:
Done